test_msgs/include/test_msgs/message_fixtures.hpp

```diff
@@ -292,6 +293,22 @@ get_messages_dynamic_array_primitives_nested()
   return messages;
 }
 
+std::vector<test_msgs::msg::DynamicArrayStaticArrayPrimitivesNested::SharedPtr>
+get_messages_dynamic_array_static_array_primitives_nested()
+{
+  std::vector<test_msgs::msg::DynamicArrayStaticArrayPrimitivesNested::SharedPtr> messages;
+
+  auto msg = std::make_shared<test_msgs::msg::DynamicArrayStaticArrayPrimitivesNested>();
+  std::vector<test_msgs::msg::StaticArrayPrimitives::SharedPtr> primitive_msgs =
+    get_messages_static_array_primitives();
+  for (size_t i = 0; i < primitive_msgs.size(); ++i)
+  {
```

Comment:
I'm somewhat surprised the linters didn't pick this up, but our usual style is curly braces on the same line as the `for` statement.

Reply:
There were no lint tests enabled for this package. I've enabled the linter and fixed issues.